__tests__/app.test.js

```diff
@@ -89,4 +87,87 @@ describe("API: /api/reviews", () => {
         });
     });
   });
+
+  describe("PATCH: /api/reviews/:review_id", () => {
+    test("201: responds with a votes incrementing by inc_vote", () => {
+      const review_id = 1;
+
+      return request(app)
+        .patch(`/api/reviews/${review_id}`)
+        .send({ inc_votes: 5 })
+        .expect(201)
+        .then(({ body: { review } }) => {
+          expect(review).toBeInstanceOf(Object);
```

Comment:
Personally I don't think you gain much by checking the data type here, you can infer this from the fact that you're checking keys with your other expect statements. 
Its not bad or wrong, I'd just have a think about what confidence you're gaining 😄